src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -37,4 +37,14 @@ public static int countEven(int[] array) {
         }
         return evenElements;
     }
+    public static int countPositive(int [] array){
```

Comment:
Please, add an additional line before the method to visually separate the methods